app/src/full/java/io/homeassistant/companion/android/settings/wear/views/SettingsWearLandingView.kt

```diff
@@ -35,44 +40,51 @@ fun SettingWearLandingView(
                 docsLink = WEAR_DOCS_LINK
             )
         }
-    ) {
+    ) { padding ->
         Column(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(all = 16.dp)
+                .padding(padding)
         ) {
-            Text(
-                text = stringResource(id = commonR.string.manage_wear_device, deviceName),
-                textAlign = TextAlign.Center
-            )
+            Row(
+                modifier = Modifier
+                    .height(48.dp)
+                    .padding(start = 72.dp, end = 16.dp),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Text(
+                    text = stringResource(id = commonR.string.manage_wear_device, deviceName),
+                    style = MaterialTheme.typography.body2,
+                    fontWeight = FontWeight.Bold,
+                    color = MaterialTheme.colors.primary
+                )
+            }
             when {
                 !hasData -> {
                     CircularProgressIndicator(modifier = Modifier.align(Alignment.CenterHorizontally))
                 }
                 isAuthed -> {
-                    Button(
-                        onClick = navigateFavorites,
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(top = 10.dp)
-                    ) {
-                        Text(stringResource(commonR.string.set_favorites_on_device))
-                    }
-                    Button(
-                        onClick = navigateTemplateTile,
-                        modifier = Modifier
-                            .fillMaxWidth()
-                            .padding(top = 10.dp)
-                    ) {
-                        Text(stringResource(commonR.string.template_tile))
-                    }
+                    SettingsRow(
+                        primaryText = stringResource(commonR.string.favorite),
```

Comment:
I know the label hasn't changed, but reviewing this as if it was new:
 - including "Set" feels unnecessary on the phone, "Favorite Entities" should be enough
 - the label for this settings doesn't match the title shown in the toolbar after tapping it (toolbar: "Wear Favorite Entities")

Reply:
Yeah I agree. changed both now to "Favorite Entities"